src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
@acasj there are too many unrelated changes in the resx file. Could you please revert them